js/apps/admin-ui/src/utils/useIsFeatureEnabled.ts

```diff
@@ -10,10 +10,12 @@ export enum Feature {
 }
 
 export default function useIsFeatureEnabled() {
-  const { profileInfo } = useServerInfo();
-  const disabledFilters = profileInfo?.disabledFeatures ?? [];
+  const { features } = useServerInfo();
 
   return function isFeatureEnabled(feature: Feature) {
-    return !disabledFilters.includes(feature);
+    return features
+      ?.filter((f) => f.enabled)
+      .map((f) => f.name)
+      .includes(feature);
```

Comment:
This code doesn't always return a `boolean`, if `features` is not defined it will return `undefined`. We can add an early return to fix this:
```suggestion
    if (!features) {
      return false;
    }

    return features
      .filter((f) => f.enabled)
      .map((f) => f.name)
      .includes(feature);
```